src/validations/class.validation.js

```diff
@@ -22,8 +21,8 @@ const updateClass = {
   }),
   body: Joi.object().keys({
     name: Joi.string(),
-    language: Joi.string(),
-    subject: Joi.string().valid(...Object.values(Subject))
+    language: Joi.string().length(2),
```

Comment:
what if there is a language with more than 2 letter ?

Reply:
I think we should use the ISO language codes, like, "en", "de", "it" etc.